src/pages/dashboard/staking/staking-pool.tsx

```diff
@@ -5,57 +5,67 @@ import RadialChart from './radial-chart';
 import globalStyles from '../../../styles/global-styles.module.scss';
 import styles from './staking-pool.module.scss';
 import classNames from 'classnames';
-import { UNKNOWN_NUMBER } from '../../../utils';
+import { images, UNKNOWN_NUMBER } from '../../../utils';
+import { Tooltip } from '../../../components/tooltip';
 
 const StakingPool = () => {
   const { dashboardState: data } = useChainData();
   const stakingPool = stakingPoolSelector(data);
 
   const currentApy = stakingPool ? round(stakingPool.currentApy, 1) : UNKNOWN_NUMBER;
-  const annualInflationRate = stakingPool ? round(stakingPool.annualInflationRate) : UNKNOWN_NUMBER;
+  const annualTotalSupplyGrowth = stakingPool ? round(stakingPool.annualTotalSupplyGrowth) : UNKNOWN_NUMBER;
   const stakedPercentage = stakingPool ? Number.parseFloat(round(stakingPool.stakedPercentage)) : undefined;
   const totalStaked = data ? formatAndRoundApi3(data.totalStake) : UNKNOWN_NUMBER;
   const stakingTargetInTokens = stakingPool ? formatAndRoundApi3(stakingPool.stakingTargetInTokens) : UNKNOWN_NUMBER;
 
   const currentApyText = currentApy === UNKNOWN_NUMBER ? currentApy : `${currentApy}%`;
-  const annualInflationRateText =
-    annualInflationRate === UNKNOWN_NUMBER ? annualInflationRate : `${annualInflationRate}%`;
+  const annualTotalSupplyGrowthText =
+    annualTotalSupplyGrowth === UNKNOWN_NUMBER ? annualTotalSupplyGrowth : `${annualTotalSupplyGrowth}%`;
+  const estimatedValueTooltip = 'Value is based on the current APR and will change in time';
```

Comment:
@bbenligiray Not sure if you meant the message in the issue "Value is extrapolated based on the current APR" literally, but I dislike the "extrapolated" word as it sounds too technical.

However, I don't have strong preference about what should the tooltip say, so feel free to insist on something.

Reply:
This is slightly better: `Value is based on the current APR and may change over time`